### PR TITLE
Update the token-bridge sub-repo

### LINKED ISSUE #17: Update token-bridge after introducing into the monorepo
- [ ] Move audit to root dir
- [ ] Move licence, code of conduct and contributing documents to root dir
- [ ] Remove Travis badge and config
- [ ] Update token-bridge/README.md with new links to licence and contributing docs
- [ ] Update contributing doc with new links
- [ ] Move gitter to main readme

### PR DESCRIPTION
**Pull request by rzadp**
Update the token-bridge sub-repo after importing into the monorepo.

Fixes #17

### CODE REVIEW

File: token-bridge/README.md [hunk 1]
```diff
@@ -282,13 +279,13 @@ When running the processes, the following commands can be used to test functiona
...
+See the [CONTRIBUTING](../CONTRIBUTING.md) document for contribution, testing and pull request protocol.
 
 ## License
 
 [![License: LGPL v3.0](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
 
-This project is licensed under the GNU Lesser General Public License v3.0. See the [LICENSE](LICENSE) file for details.
+This project is licensed under the GNU Lesser General Public License v3.0. See the [LICENSE](../LICENSE) file for details.
 
 ## References
```
Comment (akolotov) [inline token-bridge/README.md]: Let's move this section to the main README?

Comment (rzadp) [reply token-bridge/README.md]: Done.

**Comment by rzadp** [comment]
> Sorry just noticed... Does it make sense to name directory `oracle` rather than `token-bridge`? Since the token bridge is actually consist of few components: contracts, oracle, UI, monitor and deployment scripts.

Makes sense for me. I made the changes.

**Comment by rzadp** [comment]
@akolotov There is some more general information in oracle/README besides references, that would make more sense in root readme. I created #23 to do this with all sub-repo readmes after they are included in the monorepo.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+# Tokenbridge
+Monorepository for the token bridge.
+
+## Structure
+
+Sub-repositories maintained within this monorepo are listed below.
+
+| Sub-repository | Description |
+| --- | --- |
+| [token-bridge](token-bridge/README.md) | Oracle responsible for listening to bridge related events and authorizing asset transfers |
```
Comment (patitonar) [inline README.md]: Here is a reference that need to be fixed to the new directory name
```suggestion
| [oracle](oracle/README.md) | Oracle responsible for listening to bridge related events and authorizing asset transfers |
```

**Pull request merged**